pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -334,7 +334,7 @@ func (am *Alertmanager) buildIntegrationsMap(receivers []*apimodels.PostableApiR
 func (am *Alertmanager) buildReceiverIntegrations(receiver *apimodels.PostableApiReceiver, tmpl *alertingNotify.Template) ([]*alertingNotify.Integration, error) {
 	integrations := make([]*alertingNotify.Integration, 0, len(receiver.GrafanaManagedReceivers))
 	for i, r := range receiver.GrafanaManagedReceivers {
-		n, err := am.buildReceiverIntegration(r, tmpl)
+		n, err := am.buildReceiverIntegration(PostableGrafanaReceiverToGrafanaReceiver(r), tmpl)
```

Comment:
this PR does not import the latest alerting module. I did that intentionally to simplify the PRs in Grafana. Once I update to the latest version of alerting module, that will change

Reply:
These methods may go away eventually in the follow-up PRs because the signature of buildReceiverIntegrations will likely change to accept the model from alerting repo